Decisions.TruBot/Api/AuthenticationResponse.cs

```diff
@@ -0,0 +1,32 @@
+using System.Runtime.Serialization;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using Newtonsoft.Json;
+
+namespace Decisions.TruBot.Api
+{
+    [DataContract]
```

Comment:
You shouldn't need DataContract unless your members are also using DataMember.
You don't need to save this, you shouldn't need DataContract.